tests/lib/muzic/prj.conf

```diff
@@ -0,0 +1,16 @@
+# CONFIG_LIB_CPLUSPLUS Dependencies
+CONFIG_NEWLIB_LIBC=y
+CONFIG_NEWLIB_LIBC_NANO=n
+
+CONFIG_MUZIC=y
+
+CONFIG_POSIX_API=y
+CONFIG_HEAP_MEM_POOL_SIZE=65536
+
+# Test dependencies
+CONFIG_ZTEST=y
+
+# default stack sizes are not enough
```

Comment:
The default stack sizes are not enough because they are being smashed in most test functions.

Any time an array is declared at function scope without the `static` keyword, it's using stack memory.

Additionally, using a variable to declare the size of an array inside of a function is called a VLA. They are generally considered very bad.

A better solution would be to make the buffer static at file scope (i.e statically allocated in neither heap or stack memory but in the .data section).